src/libs/actions/IOU.js

```diff
@@ -152,6 +152,15 @@ function rejectTransaction({
         });
 }
 
+/**
+ * Sets IOU'S selected currency
+ *
+ * @param {Object} params.selectedCurrencyCode
```

Comment:
I'm not sure that it makes sense to pass in an object, rather than just the `selectedCurrencyCode` String here. IMO the caller should NOT know how the Onyx data is set, but it should simply pass in the currency String and delegate that responsibility to the function.

Reply:
Done!